src/crimson/os/seastore/extentmap_manager/btree/extentmap_btree_node.h

```diff
@@ -0,0 +1,99 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+
+#pragma once
+
+#include "crimson/common/log.h"
+#include "crimson/os/seastore/seastore_types.h"
+#include "crimson/os/seastore/transaction_manager.h"
+#include "crimson/os/seastore/extentmap_manager.h"
+
+namespace crimson::os::seastore::extentmap_manager{
+
+struct ExtMapNode : LogicalCachedExtent {
+  TransactionManager* tm = nullptr;
+  using ExtMapNodeRef = TCachedExtentRef<ExtMapNode>;
+  depth_t depth = 0;
```

Comment:
My pr https://github.com/ceph/ceph/pull/36148 has an updated version of FixedKVNode which includes a meta field you can use to persist depth within the nodes, you probably want to rebase and use that.